lib/wordmove/list.rb

```diff
@@ -0,0 +1,70 @@
+module Wordmove
+  class List
+    class << self
+      def start(cli_options)
+        new(cli_options).start
+      end
+    end
+
+    def initialize(options)
+      @logger = Logger.new(STDOUT).tap { |l| l.level = Logger::INFO }
+      @movefile = Wordmove::Movefile.new(options[:config])
+      @remote_vhosts = []
+      @local_vhost = []
+    end
+
+    def start
+      contents = parse_movefile(movefile: movefile)
+      generate_vhost_list(contents: contents)
+      output
+    end
+
+    def output_string(vhost_list:)
```

Comment:
Oh, thanks for your pointing out!
I marked this method as a public because I'd like to confirm its behavior through the test code.
Of course, there is no need to be public if your review had done.